packages/redux/src/bags/__tests__/selectors.test.ts

```diff
@@ -201,14 +226,14 @@ describe('bags redux selectors', () => {
       expect(selectors.getBagItemsUnavailable(mockState)).toHaveLength(0);
     });
 
-    it('should return null if there is no bag loaded', () => {
+    it('should return empty array if there is no bag loaded', () => {
```

Comment:
```suggestion
    it('should return an empty array if there is no bag loaded', () => {
```

Reply:
Here isn't better to expect a `.toEqual([])`? Just to be according to the test description?